CHANGES.md

```diff
@@ -15,7 +15,19 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- THREDDS: provide service information page details
+
+  - Add multiple metadata variables
+    (`THREDDS_ORGANIZATION_[...]`, `THREDDS_SUPPORT_[...]`, `THREDDS_ABSTRACT` and `THREDDS_KEYWORDS`)
+    allowing customization of the THREDDS server information page.
+  - Add a cross-reference to the `service-config.json` to the service information `/thredds/info/serverInfo.html` page.
+  - Resolve the Magpie `ServiceTHREDDS` configuration disallowing access
+    to `/twitcher/ows/proxy/thredds/info/serverInfo.html` by default.
+    Every content under the THREDDS `/info/` prefix will be considered a Magpie `BROWSE` permission of "metadata".
+    It is up to the organization to make this endpoint visible, either using `optional-components/all-public-access`
```

Comment:
forgot to update `all-public-access`?  Just raise a concern, maybe it's not necessary.

Reply:
This will be affected by the browse permissions set by all-public-access. I don't think it needs a specific entry just for the info route.

Reply:
Not necessary. What is important is to have "some `BROWSE` permission" defined either recursively on the service, or onto the specific `serverInfo.html` resource. I referenced `all-public-access` because it already sets this `BROWSE` as an example, but didn't want to impose a specific dependency.